numpyro/examples/datasets.py

```diff
@@ -313,9 +326,18 @@ def iter_dataset(dset, batch_size=None, split="train", shuffle=True):
 
 
 def load_dataset(
-    dset, batch_size=None, split="train", shuffle=True, num_datapoints=None
+    dset,
+    batch_size=None,
+    split="train",
+    shuffle=True,
+    num_datapoints=None,
+    subset_key=None,
```

Comment:
Would it be simpler to use `split` for this?
```
def _load_9mers():
    _download(NINE_MERS)
    file_path = os.path.join(DATA_DIR, "9mers_data.pkl")
    return pickle.load(open(file_path, "rb"))
```

Reply:
Makes sense, and is much simpler. I'll update it.